js/packages/teams-ai/src/actions/SayCommand.spec.ts

```diff
@@ -152,7 +152,7 @@ describe('actions.sayCommand', () => {
             const citations = [
                 {
                     title: 'the title',
-                    url: '',
+                    url: 'http://example.com',
```

Comment:
Not sure how this test passed previously - [`SayCommand.ts` for citations](https://github.com/microsoft/teams-ai/blob/release/v1/js/packages/teams-ai/src/actions/SayCommand.ts) sets `url` to `citation.url` or `undefined`, so an empty string would return falsy.